src/components/InstructionsModal/InstructionsModal.tsx

```diff
@@ -8,7 +8,7 @@ export const InstructionsModal = ({
   onCancel,
 }: {
   title: string,
-  children: React.ReactNode,
+  children: string,
```

Comment:
cant it it be both?
```suggestion
  children: React.ReactNode|string,
```

Reply:
This will go directly to `__html` and I can't see how react nodes can be converted there. We can use them directly though, but that's definitely not the case now. So I'd leave it, because this type reflects both utilization and invocation of this param.